src/components/input/select/style.css

```diff
@@ -132,7 +134,11 @@
 }
 
 :host>div.options::slotted(smoothly-item) {
-	padding: .7em;
+	padding: .7em .7em .7em calc(.7em - 5px);
```

Comment:
Why use pixels?

Reply:
Because that thick border-left that shows if an item is selected is 5px wide!

Reply:
...and now i have replaced that pixel-value with em in both smootlhy-item and in smoothly-input-select!